documentation/src/main/wiki/HttpServiceAdapter.md

```diff
@@ -208,3 +229,64 @@ In response, external service returns:
 ```
 
 which is finally wrapped into [Adapter Response](#adapter-response).
+
+##### Setting service query parameters
+We can use the `queryParams` JSON object to define the service query parameters and their values directly from template. Consider the following service configuration:
+
+```json
+  "config": {
+    "address": "knotx.knot.service",
+    "services": [
+      {
+        "name" : "products",
+        "address" : "knotx.adapter.service.http",
+        "params": {
+          "path": "/service/products/"
+        }
+      }
+    ]
+  }
+```
+
+We can set query parameters sent to the service using the following snippet:
+
+```html
+<script data-knotx-knots="services,handlebars" type="text/knotx-snippet"
+    data-knotx-service="products"
+    data-knotx-params='{"queryParams":{"amount":"4"}}">
```

Comment:
It looks that something is wrong in this line. `'` opens the params, but it is closed with `"`.